src/LintingViewExtension/LinterViewModel.cs

```diff
@@ -144,7 +134,7 @@ private void AddNewNodeIssue(string issueNodeId, string ruleId)
             var newIssue = new NodeRuleIssue(ruleId, GetLinterRule(ruleId) as NodeLinterRule);
             newIssue.AddAffectedNodes(new List<NodeModel> { issueNode });
 
-            NodeIssues.Add(newIssue);
+            this.dispatcher.Invoke(() => { NodeIssues.Add(newIssue); });
```

Comment:
Should this.GraphIssues.Clear() & this.NodeIssues.Clear() be on dispatcher as well ?

Reply:
Yes will do